source/gloperate-qtquick/source/TextureRenderItem_ogl.cpp

```diff
@@ -65,7 +65,7 @@ void TextureRenderItem::render()
 
         if (slot && slot->type() == typeid(globjects::Texture *))
         {
-            m_texture = static_cast< Slot<globjects::Texture *> * >(slot)->value();
+            m_texture.reset(static_cast< Slot<globjects::Texture *> * >(slot)->value());
```

Comment:
This takes ownership of the texture. In general, if ownership is transferred, this should be done by move-assigning a unique_ptr.

In this case, the texture is owned by the stage that created it, thus a simple (non-owning) raw pointer should be used here.